manifests/init.pp

```diff
@@ -162,6 +165,16 @@
     $iftemplate = template('network/ifcfg-eth.erb')
   }
 
+  if $flush {
+    exec { 'network-flush':
+      user        => 'root',
+      command     => "/usr/sbin/ip addr flush dev ${interface}",
```

Comment:
@razorsedge Good point Mike, what would you suggest, smth. like `path   => '/sbin:/usr/sbin',`?